lib/cmds/space_cmds/use.ts

```diff
@@ -31,7 +32,14 @@ module.exports.builder = yargs => {
     .epilog('Copyright 2019 Contentful')
 }
 
-function showSuccess(space, env) {
+interface Space {
```

Comment:
Can we use `@contentful/types` maybe?

Reply:
I'll take a look into it.

Reply:
Had to import from `contentful-management`, because it seems like the types are different, from this package and the `@contentful/types` 🤔

Reply:
Sounds very likely 😞